include/votca/tools/eigen.h

```diff
@@ -51,6 +51,7 @@
 #include <Eigen/Eigen>
 #include <unsupported/Eigen/CXX11/Tensor>
 #include <unsupported/Eigen/FFT>
+#include <unsupported/Eigen/SparseExtra>
```

Comment:
I am really not a huge fan to incl. all the _unsupported_ headers by default, could we just incl this header in `src/libtools/eigenio_matrixmarket.cc` only?

Reply:
on the one hand I agree, I have been arguing with myself over this, but then I have to add all the #pragma stuff and warning stuff again. I will experiment a bit.

Reply:
Yes, I see, in some sense the header aren't optional, so it doesn't matter to much were we included, they have to be there anyhow.